index.html

```diff
@@ -0,0 +1,145 @@
+<!DOCTYPE html>
```

Comment:
I can see that you started again in another file. It would be nice that (if you do this again in future) that you delete the old file. The reason being is we try to keep pull requests as small as possible. Obviously in this case we have asked you to submit all of your work as a pull request, so it's going to be big - however, the `index.html` here isn't needed as you've restarted with the `twitter.html`